addon/components/file-dropzone/component.js

```diff
@@ -39,15 +39,15 @@ const dragListener = new DragListener();
         {{#if dropzone.supported}}
           Drag and drop images onto this area to upload them or
         {{/if}}
-        {{#file-upload name="photos"
-                      accept="image/*"
-                      multiple=true
-                      onfileadd=(action "uploadImage")}}
+        <FileUpload name="photos"
+                    accept="image/*"
+                    multiple=true
+                    @onfileadd={{action "uploadImage"}}>
```

Comment:
The coding style looks different than what I would expect. I'm used to this one:

```hbs
<FileUpload
  @onfileadd={{action "uploadImage"}}
  name="photos"
  accept="image/*"
/>
```

So it's basically about this rules:

- If there are multiple lines, each attribute or property should be on a new line
- It should be intended by two spaces.
- Arguments should be before HTML attributes.

I thought that's also what's used in Ember docs but noticed that Ember docs aren't consistent on these ones.

Reply:
I usually do html attributes first and then args, usually actions last

Reply:
Hmm I wonder if there is any [ember-template-lint rule](https://github.com/ember-template-lint/ember-template-lint) about this...

Reply:
@mcfiredrill I am glad you asked 😄 
https://github.com/ember-template-lint/ember-template-lint/pull/871
I requested this a while ago :)
https://github.com/ember-template-lint/ember-template-lint/issues/684

Although it isn't exactly what @knownasilya mentioned